README.md

```diff
@@ -1,2 +1,104 @@
 # white-out
-Transform stream to censor object data.
+
+A transform stream used to censor data from objects before passing them down the pipeline.
```

Comment:
I thought we traditionally did all the badges right after the module name. Then started talking about it.